packages/graphback-datasync/src/providers/DatasyncKnexDataProvider.ts

```diff
@@ -0,0 +1,42 @@
+import { GraphQLObjectType } from 'graphql';
```

Comment:
I will keep it mongo only for the moment. 
We will focus on Mongo and bring knex at later point